app/assets/javascripts/discourse/app/components/composer-container.hbs

```diff
@@ -98,7 +99,7 @@
             @toggleComposer={{this.composer.toggle}}
             @toggleToolbar={{this.composer.toggleToolbar}}
             @toggleFullscreen={{this.composer.fullscreenComposer}}
-            @disableTextarea={{this.composer.disableTextarea}}
+            {{!-- @disableTextarea={{this.composer.disableTextarea}} --}}
```

Comment:
The failures are related to these two commented out arguments

Reply:
No idea why i commented that out to begin with – classic, thanks